src/splits.rs

```diff
@@ -299,6 +299,318 @@ pub enum Split {
+    ///
+    /// Splits after unlocking High Halls Ventrica
+    UnlockedHangTube,
+    /// Unlocked Songclave (Ventrica)
+    ///
+    /// Splits after unlocking Songclave Ventrica
+    UnlockedEnclaveTube,
+    /// Unlocked Memorium (Ventrica)
+    ///
+    /// Splits after unlocking Memorium Ventrica
+    UnlockedArboriumTube,
+    // endregion: Ventricas
+
+    // region: ShakraEncounters
+    /// Seen Shakra Bonebottom (Mapper)
+    ///
+    /// Splits after seeing Shakra in Bonebottom
+    SeenMapperBonetown,
+    /// Seen Shakra Marrow (Mapper)
+    ///
+    /// Splits after seeing Shakra in Marrow
+    SeenMapperBoneForest,
+    /// Seen Shakra Deep Docks (Mapper)
+    ///
+    /// Splits after seeing Shakra in Deep Docks
+    SeenMapperDocks,
+    /// Seen Shakra Far Fields (Mapper)
+    ///
+    /// Splits after seeing Shakra in Far Fields
+    SeenMapperWilds,
+    /// Seen Shakra Wormways (Mapper)
+    ///
+    /// Splits after seeing Shakra in Wormways
+    SeenMapperCrawl,
+    /// Seen Shakra Greymoor (Mapper)
+    ///
+    /// Splits after seeing Shakra in Greymoor
+    SeenMapperGreymoor,
+    /// Seen Shakra Bellhart (Mapper)
+    ///
+    /// Splits after seeing Shakra in Bellhart
+    SeenMapperBellhart,
+    /// Seen Shakra Shellwood (Mapper)
+    ///
+    /// Splits after seeing Shakra in Shellwood
+    SeenMapperShellwood,
+    /// Seen Shakra Hunter's March (Mapper)
+    ///
+    /// Splits after seeing Shakra in Hunter's March
+    SeenMapperHuntersNest,
+    /// Seen Shakra Blasted Steps (Mapper)
+    ///
+    /// Splits after seeing Shakra in Blasted Steps
+    SeenMapperJudgeSteps,
+    /// Seen Shakra Sinner's Road (Mapper)
+    ///
+    /// Splits after seeing Shakra in Sinner's Road
+    SeenMapperDustpens,
+    /// Seen Shakra Mount Fay (Mapper)
+    ///
+    /// Splits after seeing Shakra in Mount Fay
+    SeenMapperPeak,
+    /// Seen Shakra Bilewater (Mapper)
+    ///
+    /// Splits after seeing Shakra in Bilewater
+    SeenMapperShadow,
+    /// Seen Shakra Sands of Karak (Mapper)
+    ///
+    /// Splits after seeing Shakra in Sands of Karak
+    SeenMapperCoralCaverns,
+    // endregion: ShakraEncounters
+
+    // region: MiscTE
+    /// Met Merchant Enclave (NPC)
+    ///
+    /// Splits after talking to Jubilana in Songclave
+    MetCityMerchantEnclave,
+    /// Met Sherma Enclave (NPC)
+    ///
+    /// Splits after talking to Sherma in Songclave
+    MetShermaEnclave,
+    /// Unlock Prince Cage (Event)
+    ///
+    /// Splits when you unlock Green Prince's Cage in Sinner's Road
+    UnlockedDustCage,
+    /// Met Green Prince Cogwork (NPC)
+    ///
+    /// Splits when you talk to Green Prince in Cogwork Dancer's arena
+    GreenPrinceLocationIsVerdania,
+    /// Seen Fleatopia Empty (Event)
+    ///
+    /// Splits when you find Fleatopias location
+    SeenFleatopiaEmpty,
+    /// Faydown Cloak (Ability)
+    ///
+    /// Splits when you obtain Double Jump
+    HasDoubleJump,
```

Comment:
In the interest of creating less work for Silksong Split Maker, I'd like all these `(Ability)` parentheticals changed to `(Skill)`.

Also, for the variant keys like `HasDoubleJump`, I'd rather they more closely match the split name, like from `Faydown Cloak` to `FaydownCloak`, when it differs from the field name